site/content/installation/installing-ngf/manifests.md

```diff
@@ -45,6 +45,8 @@ kubectl apply -f https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric
 
 {{<note>}}By default, NGINX Gateway Fabric is installed in the **nginx-gateway** namespace. You can deploy in another namespace by modifying the manifest files.{{</note>}}
 
+{{<note>}}If you are deploying NGINX Gateway Fabric in an Openshift environment, we've supplied a baseline [SecurityContextConstraints](https://raw.githubusercontent.com/nginxinc/nginx-gateway-fabric/v1.2.0/deploy/manifests/scc.yaml) manifest that you can download, modify if needed, and apply. You will also need to make sure the SecurityContextConstraints resource is added to the ClusterRole rbac.{{</note>}}
```

Comment:
Unsure of this. Had a couple of choices I was considering.

1. Don't even mention the scc.yaml manifest we added.
2. Mention it here in the regular manifest installation document with light instruction on how to use it.
3. Create a different document which is specific to installation on Openshift with manifests and explain this scc.yaml in further detail and include details on how to specifically modify the ClusterRole to allow for the SecurityContextConstraints to be added. 

I chose to proceed with Option 2.